agent/bench-scripts/postprocess/BenchPostprocess.pm

```diff
@@ -10,12 +10,16 @@ use warnings;
 use File::Basename;
 use Cwd 'abs_path';
 use Exporter qw(import);
-use List::Util qw(max);
+use List::Util qw(max sum);
 use JSON;
 
-our @EXPORT_OK = qw(value_exists trim_series get_label create_uid get_length get_uid get_mean remove_timestamp get_timestamps write_influxdb_line_protocol get_cpubusy_series calc_ratio_series calc_sum_series div_series calc_aggregate_metrics calc_efficiency_metrics create_graph_hash get_json);
+our @EXPORT_OK = qw(
+	calc_aggregate_metrics calc_efficiency_metrics calc_ratio_series
+	calc_sum_series convert_samples_hash_to_array create_graph_hash
+	create_uid get_cpubusy_series get_json get_label get_length
+	get_mean_hash get_uid);
 
-my $script = "BenchPostprocess";
+my $script = "BenchPostprocess";  # FIXME:  this initialization doesn't seem to happen....
```

Comment:
Should we track this in an issue?

Reply:
That's an interesting question.

In light of other conversations that we've had today, rather than draw resources and attention away from other endeavors, I'm inclined to ignore this mystery.

FWIW, `$script` is only referenced on the error paths of `get_cpubusy_series()`, which is somewhat less used than most functions in this module.  Moreover, I'm not up on Perl's scoping rules, but it's possible that the references to it will be satisfied by definition in the calling module rather than the containing one, in which case the lack of initialization here is unimportant.

If you like, I could revert this change....  🙂

Reply:
No further changes needed for this.